6_Model Comparison.ipynb

```diff
@@ -0,0 +1,6627 @@
+{
```

Comment:
<p>I would add a comment here as to why it is you (and the PyMC3 people) are using 10^6 as a prior on precision.</p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Added the following explanation:
 "We take an uninformative prior with a high standard deviation, which is close to a uniform distribution. This effectively means that we are taking no assumption on the parametrization."